.github/scripts/validate_binaries.sh

```diff
@@ -41,5 +41,8 @@ else
         fi
 
         python  ./test/smoke_test/smoke_test.py
+
+        conda deactivate
```

Comment:
Should these be aligned with the else, i.e. directly above the fi?

Reply:
Oh, nvm. It was only created inside this else statement. So it should be correct.